test/e2e/example_k8petstore.go

```diff
@@ -150,7 +150,7 @@ T:
 	Ω(totalTransactions).Should(BeNumerically(">", minExpected))
 }
 
-var _ = Describe("[Example] Pet Store [Skipped]", func() {
+var _ = Describe("Pet Store [Feature:Example]", func() {
```

Comment:
ah, I was confused, because https://github.com/ihmccreery/kubernetes/blob/feature-example/hack/jenkins/e2e.sh doesn't show `[Feature:]`, but I guess that's because it's a branch. OK.